styles.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "myFont";
+  src: url(http://www.anaveer.in/honda/font/MYRIADPRO-REGULAR.OTF);
+ }
+ 
+body {
+  margin: 0;
+ }
+
+.main-container{
+  border-top: 5px solid #FFA500;
+  background-color: #FFF;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+ }
+
+.vertical-separator {
+  background: linear-gradient(to top, transparent, orange, transparent);
+  margin: 100px 40px;
+  width: 5px;
+  height: 250px;
+ }
+ 
+div { 
+  display: flex ;
+  margin: 0;
+ }
+ 
+.mountain-image{
+  background-repeat: repeat-x;
+  background-image: url("/home/natashamartinelli/Documentos/training maquetado/triangle.png");
+  height: 12px;
+ }
+ 
+.second-container {
+  background-color: #EEEEEE;
```

Comment:
that colour could be written like this: ```#EEE``` in fact all colours with the format #aabbcc can be written like #abc